migtests/tests/oracle/assessment-report-test/cleanup-db

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+source ${SCRIPTS}/functions.sh
+
+echo "Deleting existing data in source database"
+run_sqlplus_as_schema_owner ${SOURCE_DB_NAME} ${TESTS_DIR}/oracle/utils/delete_full_schema
+
+# Re-running the script since all dependent objects are not dropped in the first run. 
```

Comment:
but if there are dependencies of objects then its not confirmed that it will be deleted in two runs or n runs.

Is `CASCADE` while dropping not working here?

Reply:
No cascade isn't working.
I thought of fixing the cleanup script separately.
Doing it in the same PR.

Reply:
yeah if possible lets do more things in a separate PR, it will be easier to review. This one is already reviewed from my side.